x/cli/app.go

```diff
@@ -79,16 +79,22 @@ func (a *App) parseArgs() ([]string, []string) {
 }
 
 func (a *App) commandContext() CommandContext {
-	cmds, flags := a.parseArgs()
+	var (
+		cmds, flags = a.parseArgs()
+		args        = make(map[string]string)
```

Comment:
```suggestion
		args        = make(argProvider)
```
?

Reply:
For the commandContext args is only a `map[string]string` that represents the data parsed, argProvider is here just to be a polyfill of provider. This type does not and should not be used everywhere